internal/exit/exit.go

```diff
@@ -24,23 +24,23 @@ import (
 	"github.com/submariner-io/submariner-operator/pkg/version"
 )
 
-// OnError will print your error nicely and exit in case of error.
-func OnError(message string, err error) {
+// OnError exits in case of error.
+func OnError(err error) {
```

Comment:
We do want to print what the err is

Reply:
With the current usages the error is already printed - here we provide a convenience to exit if non-nil. If the caller wants to print the error they can call `WithMessage`.